### ISSUE TITLE
The `validateMatch` helper should support more specific error messages

### ISSUE THREAD

**Issue by erikphansen** [usds/us-forms-system#268]
**Is your feature request related to a problem? Please describe.**
It should be possible to show the user more specific error messages when using the [`validateMatch` helper](https://github.com/usds/us-forms-system/blob/master/lib/js/validation.js#L373). Instead of simply showing: "Please ensure your entries match" it would be nice to show a message like "Please ensure your email entries match". This is [being introduced in `vets-website`](https://github.com/department-of-veterans-affairs/vets-website/pull/8416/files/b6cdbc82bcfedd24ff3211eab9ede07babdb6e56#diff-e364e8f9f088cf53b6804cd8040f260aR257).

**Describe the solution you'd like**
Optionally allow to make the error message more specific.

**Describe alternatives you've considered**

**Additional context**

**Comment by erikphansen** [comment]
I'm going to create a PR that addresses this that you are more than welcome to close without merging. It will be purely additive and backward compatible, of course.

**Comment by dmethvin-gov** [comment]
Yeah we should definitely make it so that these strings can be changed, especially for internationalization. 👍

**Comment by dmethvin-gov** [comment]
Yes, I would have it pass in the entire error message, especially since the form of the message is not dependent on data the user entered.

**Comment by jcmeloni-usds** [comment]
Agree on usefulness!  Putting in Phase II for the moment just as an organizational move.
